ansible/group_vars/staging/vars

```diff
@@ -0,0 +1,6 @@
+---
+env: staging
+infinite_image_version_tag: development
```

Comment:
I suppose this might answer my earlier question about deploying different image tags, feels a little less discoverable than passing it in the bash script as somone less familiar with ansible, but probably a better way to do it